<?xml version="1.0"?>


<launch>
  <node name="map_server" pkg="map_server" type="map_server" args="$(find navi)/waypoint_files/Free.yaml">
  <param name="frame_id" value="odom" />
  </node>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    
    <rosparam file="$(find navi)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find navi)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find navi)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find navi)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find navi)/config/base_local_planner_params.yaml" command="load" /> 
    <rosparam file="$(find navi)/config/move_base_params.yaml" command="load" />
    
    <param name="base_global_planner" type="string" value="navfn/NavfnROS" />
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>

    <remap from="odom" to="/outdoor_nav/odometry/EKF_estimated" />
    <remap from="map" to="/map"/>
    <remap from="cmd_vel" to="/cmd_vel" />
    <remap from="scan" to="/scan" />
  </node>

</launch>


